README.md

```diff
@@ -1,11 +1,130 @@
-# amrita_pyq
+![Static Badge](https://img.shields.io/badge/ampyq-darkred?logoColor=white)
+![Go Badge](https://img.shields.io/badge/Go-00ADD8?logo=go&logoColor=white)
+![Last Updated](https://img.shields.io/badge/Last%20Updated-December-00ADD8?style=flat&logoColor=white&labelColor=5E5D5D&color=D0A93E)
 
-## Contributing
 
-We welcome contributions! Please see our [Contributing Guide](.github/CONTRIBUTING.md) for more details.
+# **Amrita PYQ CLI Tool Documentation**
+<p align="center">
+  <img src="https://github.com/user-attachments/assets/44822d21-0e1d-4b3e-baa3-3450b5cc14fc" alt="amritalogo" width="550" height="200">
+</p>
 
-<!-- 
-## Code of Conduct
+## **Overview**
```

Comment:
When `##` makes it bold, why `**` again? Same below too

Reply:
To add significance to the heading, I can delete it if needed....

Reply:
I feel it's too much.

Reply:
okay then, will revert it